Mathlib/Algebra/QuadraticDiscriminant.lean

```diff
@@ -15,7 +15,7 @@ This file defines the discriminant of a quadratic and gives the solution to a qu
 
 ## Main definition
 
-- `discrim a b c`: the discriminant of a quadratic `a * x * x + b * x + c` is `b * b - 4 * a * c`.
+- `discrim a b c`: the discriminant of a quadratic `a * (x * x) + b * x + c` is `b * b - 4 * a * c`.
```

Comment:
I don't think these parentheses (and those below) are adding any clarity.

Reply:
They're not adding *clarity*, changing the *associativity*.